docs/FAQ.md

```diff
@@ -33,15 +41,15 @@ If all went well, `multicast` is now installed and working :tada:
 
 ```bash
 # cd /MY-AWESOME-DEV-PATH
-python3 -m multicast HEAR --use-std --port 59595 --join-mcast-groups 224.0.0.1 --bind-group 224.0.0.1
+python3 -m multicast --daemon HEAR --use-std --port 59595 --group 224.0.0.1
 ```
 
 Caveat: `RECV` is much more useful if actually used in a loop like:
 
 ```bash
 # cd /MY-AWESOME-DEV-PATH
 while true ; do  # until user Ctrl+C interrupts
-python3 -m multicast RECV --use-std --port 59595 --join-mcast-groups 224.0.0.1 --bind-group 224.0.0.1
+python3 -m multicast RECV --use-std --port 59595 --group 224.0.0.1 --groups 224.0.0.1
 done
```

Comment:
the change here is a bit complex, but basicly the `--join-mcast-groups` became `--groups` and was moved to the end of the arguments, while the `--bind-group` became `--group` and "stayed put" resulting in the final order when considering the move of `--groups`

the use of `RECV` here is to emphasize the lack of `--daemon` which is the real difference, this is also the only real-world use-case I could come up with for the `RECV` sub-command as this is how it would be used in a prototyping case. 

The finer details of the two commands `HEAR` and `RECV` belong in the usage document.